examples/simple_env/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+// Any changes to this file likely necessitate changes to the project's README.
+//go:generate goclap -type mycli
+
+import (
+	"fmt"
+	"os"
+)
+
+// Print a string with the option to make it uppercase.
+type mycli struct {
```

Comment:
I don't think this example should include command line options. It takes away from the purpose to show an environment variable example.